docs/DeveloperGuide.md

```diff
@@ -602,29 +602,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a … | I want to … | So that I can…|
 | - | - | - | - |
-| `* * *` | new user | see usage instructions | refer to instructions if I forget how to use the app |
 | `* * *` | confused user | have a help command | learn how to use the application |
-| `* * *` | user | add a new person | |
-| `* * *` | user | add issues | |
-| `* * *` | user | see a list of pending tasks sorted by deadline | prioritise which to do |
-| `* * *` | user | search for tasks | |
+| `* * *` | user | add issues | track associated issues with rooms and their status|
+| `* * *` | user | see a list of pending issues sorted by deadline | prioritise which issue to work on |
+| `* * *` | user | search for issues | find a specific task and get its information or tasks matching a specific criteria|
 | `* * *` | user | close issues | focus on open issues |
-| `* * *` | user | update issues | |
-| `* * *` | user | remove issues | |
-| `* * *` | user | add residents | |
-| `* * *` | user | search for residents | |
+| `* * *` | user | update issues | ensure it has the latest and correct information |
+| `* * *` | user | remove issues | not worry about issues that are no longer relevant |
+| `* * *` | user | add residents | track and manage residents who are staying or want to stay in the hostel |
+| `* * *` | user | search for residents | find a specific resident's information or residents matching a specific criteria |
```

Comment:
Change to: 
| * * * | user | search for residents | find a a specific resident's information |
Can remove  " .... residents matching a specific criteria". 

Rationale:
rfind only supports searching based on name. (can't find email etc)

Reply:
Okie done, thanks